### ISSUE TITLE
Audio quality

### ISSUE THREAD

**Issue by Juke34** [acidhax/chromecast-audio-stream#65]
Hi, 
I feel a huge difference in audio quality using this app according to what I can hear using the cast from the google chrome .
Is there away to check what is send through your app (qualily) ?

Best regards

**Comment by johnboyleboy** [comment]
I agree, the audio quality is just way too low. Bass is more or less non existent. I appreciate the effort though, the app is a great idea if it gets implemented correctly.

**Comment by SaTiaN** [comment]
@Juke34 @johnboyleboy 

Hey! I am happy to say that I had the same problem but solved it!

If you have different sound devices installed on your system, try switching to another default device (close the application and restart it before changing).

On my system, the default "speakers" Realtek High Definition Audio, was  hardly outputting any bass, but my "Digital Output" Realtek High Definition Audio (which normally has no sound output on my system) gave me the real deal.

Hope this works!

**Comment by johnboyleboy** [comment]
@SaTiaN So you're suggesting sacrificing my laptop speakers for only when I have the chromecast available? I don't want to have to enable/disable my default audio driver in order to use my chromecast.

**Comment by SaTiaN** [comment]
@johnboyleboy 

hehe no.  On windows it's quite simple to change which device you want to be default:

Right click on the sound icon in the system tray, select "playback devices", select the preferred audio output device and click "set default".   Just change this around whenever you aren't using the chromecast or whatever...
![1](https://cloud.githubusercontent.com/assets/11357584/21289785/cd6e8ba4-c4a5-11e6-8992-061d531e318e.JPG)
![2](https://cloud.githubusercontent.com/assets/11357584/21289786/cd744a94-c4a5-11e6-91fa-cd2a45fb1e13.jpg)
![3](https://cloud.githubusercontent.com/assets/11357584/21289787/cd767864-c4a5-11e6-99d4-a154bf4ffb19.JPG)

**Comment by s4n3** [comment]
I can't find the "Realtek Digital Output" option.... 
Operating : Win 7
I have ticked the " Show Disabled Devices" but I only see the "speakers"
